backend/ts/src/main/scala/aqua/backend/ts/TypeScriptFunc.scala

```diff
@@ -68,7 +68,7 @@ case class TypeScriptFunc(func: FuncCallable) {
        |    const promise = new Promise<$retType>((resolve, reject) => {
        |        request = new RequestFlowBuilder()
        |            .disableInjections()
-       |            .withTTL($configArgName?.ttl || 5000)
+       |            .withTTL($configArgName?.ttl || 7000)
```

Comment:
sure? why?

Reply:
Replaced with an if statement which uses the default value from JS SDK